src/Cmixin/Holidays/gb-scotland.php

```diff
@@ -1,36 +1,34 @@
 <?php
 
 return array(
-    'new-year'      => function ($year) {
-        $date = DateTime::createFromFormat('U', (string) strtotime("$year/01/01"));
+    'new-year' => function ($year) {
+        $date = new DateTime("$year-01-01");
 
-        if (date('N', strtotime("$year/01/01")) >= 6) {
-            $date->add(new DateInterval('P2D'));
+        if ($date->format('N') > 5) {
+            $date->modify('+2 days');
```

Comment:
Does that mean that if the new year is a Sunday, the holiday fall the Tuesday? Should'nt it be `'next monday'` too?

Reply:
In Scotland it does as they have a bank holiday on the 2nd January, so New Year would fall on the Tuesday.

Reply:
You can see the Scottish bank holidays here: https://www.gov.uk/bank-holidays#scotland

2017 has an example of New Year falling on the Tuesday instead of the Monday.